docs/conf.py

```diff
@@ -65,6 +70,46 @@
 # (note that `group` is the GitHub user or organization)
 issues_github_path = "skops-dev/skops"
 
+
+def linkcode_resolve(domain, info):
+    if domain != "py":
+        return None
+    if not info["module"]:
+        return None
```

Comment:
this is checked bellow again (`info.get("module")` is safer than `info["module"]` since the latter can raise an exception if the key doesn't exist.

Reply:
✅ Done

Reply:
Now it works, I updated the PR. Could you please take a look at it? Thank you so much for everything!